src/lambdas/dispatcher/src/lambda_function.py

```diff
@@ -87,6 +99,30 @@ def _analysis(command: AnalysisCommand, client: DataApiClient) -> List[Dict[str,
     return jobs
 
 
+def _version_update(command: VersionUpdateCommand):
+    ds = command.parameters.dataset
+    # TODO: Put datasets/functions in a dictionary?
+    if ds == UpdatableDatasets.wur_radd_alerts:
```

Comment:
as stated above, I would just make parametrize the subclasses better and then you just create a VersionUpdateJob with the passed parameters, instead of checking each one